packages/common/react-components/src/components/Searchbar/Searchbar.tsx

```diff
@@ -25,7 +24,20 @@ export const Searchbar: FC<{
   return (
     <div className='flex flex-1 flex-col bg-white'>
       <div className={mx('flex flex-1 items-center p-2', border && 'border rounded')}>
-        <Input className='w-full outline-0' spellCheck={false} onChange={handleChange} placeholder='Search...' />
+        <Input
```

Comment:
`react-components` generally don't use each other since they’re meant to stand on their own. Since this is combining `Input` and `Button` elements, I think it probably belongs in `react-appkit`.